### PR TITLE
Styles for new CMS features

### PR DESCRIPTION
**Pull request by noahmanger**
Because the `.rich-text` class is automatically applied to all blocks
from the CMS (with no way to remove it), this is required so that
nested lists display correctly.

---

## Update:
This pull request morphed as I did more work on the CMS. This now includes styles for the latest update posts and feed:

Post:
image

Feed filter:
image

### CODE REVIEW

**Comment by noahmanger** [comment]
Updated the initial comment to explain what this is doing now that I committed some more changes.

File: scss/components/_tags.scss [hunk 1]
```diff
@@ -18,6 +18,24 @@
...
+  border-radius: 4px;
+  color: $primary;
+  display: inline-block;
+  font-family: $sans-serif;
+  letter-spacing: -.3px;
+  padding: u(0 .5rem 0 .75rem);
+  margin: u(.5rem 3rem 0 0);
+}
+
+.tag--secondary {
```
Comment (ccostino) [inline scss/components/_tags.scss]: Was this meant to be a double underscore instead?

File: scss/modules/_hero.scss [hunk 2]
```diff
@@ -69,6 +69,12 @@
   }
 }
 
+.hero--press {
```
Comment (ccostino) [inline scss/modules/_hero.scss]: Same here - or is there a convention I'm missing?

File: scss/components/_tags.scss (hunk 1 above)
Comment (noahmanger) [inline scss/components/_tags.scss]: Nope. The convention is `.base--modifier` and `.base__element` . So if it's just a variant, as in this case, of the same type of element, then it's double dash.

Comment (ccostino) [inline scss/components/_tags.scss]: Thanks!

**Comment by ccostino** [comment]
Looks good, thanks @noahmanger!

**Pull request merged**